src/test/java/core/basesyntax/tests/TestsModels.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import org.junit.Test;
+
+public class TestsModels {
```

Comment:
tests are created to check some logic - so this test is redundant and can better be removed
in case of any issues with coverage you can exclude `model` package in pom.xml